cmd/influxd/launcher/cmd.go

```diff
@@ -591,20 +586,6 @@ func (o *InfluxdOpts) BindCliOpts() []cli.Opt {
 			Desc:  "The maximum number of group by time bucket a SELECT can create. A value of zero will max the maximum number of buckets unlimited.",
 		},
 
-		// NATS config
-		{
-			DestP:   &o.NatsPort,
-			Flag:    "nats-port",
-			Desc:    fmt.Sprintf("Port that should be bound by the NATS streaming server. A value of %d will cause a random port to be selected.", nats.RandomPort),
-			Default: o.NatsPort,
-		},
-		{
-			DestP:   &o.NatsMaxPayloadBytes,
-			Flag:    "nats-max-payload-bytes",
-			Desc:    "The maximum number of bytes allowed in a NATS message payload.",
-			Default: o.NatsMaxPayloadBytes,
-		},
```

Comment:
Do we want to do anything with these flags to allow for them to be in existing scripts and not cause problems?